AzureCalling/app/src/main/java/com/azure/samples/communication/calling/external/calling/CallingContext.java

```diff
@@ -216,6 +220,23 @@ public CompletableFuture turnOffVideoAsync() {
                 call.stopVideo(appContext, localVideoStream).thenRun(() -> cameraOn = false));
     }
 
+    public CompletableFuture<LocalVideoStream> switchCameraAsync() {
+        return getLocalVideoStreamCompletableFuture().thenCompose(localVideoStream -> {
+            final VideoDeviceInfo currentCamera = localVideoStream.getSource();
+            localVideoStreamCompletableFuture = new CompletableFuture<>();
+            final LocalVideoStream newVideoStream;
+            if (currentCamera.getCameraFacing().name().equalsIgnoreCase(CameraType.FRONT.name())) {
+                newVideoStream = new LocalVideoStream(getBackCamera(), appContext);
+            } else {
+                newVideoStream = new LocalVideoStream(getFrontCamera(), appContext);
+            }
```

Comment:
Not sure if Android Calling SDK has the `.switchSource` method for `LocalVideoStream`. In iOS we're able to leverage `localVideoStream.switchSource(camera:completionHandler:)` to perform camera switching without the need to set a new LocalVideoStream instance

Reply:
Thanks, I see Android SDK has the same thing.  I've implemented this functionality now in the latest version.